src/file/file.service.ts

```diff
@@ -78,16 +78,24 @@ export class FileService {
       fullName,
       ownerID,
       _id: id,
+      isDeleted: false,
       parent: parentID
     });
 
     return await FilesRepository.create(file);
   }
 
+  // recursive
   public static async delete(fileId: string): Promise<void> {
     // TODO: Delete all children
+    const file: IFile = await this.getById(fileId);
+    if (file.type === 'Folder') {
```

Comment:
`file.type === 'Folder'` wtf? Why isn't it a constant and why is it named like this?
Also remove // TODO comment...